### PR TITLE
feat: add report meta info

### PR DESCRIPTION
**Pull request by li-jin-gou**
- zh:  提供 `ReportMetaInfo` 方法用于上报 `opensergo` 元信息
- en: provide `ReportMetaInfo` method for reporting `opensergo` meta info

### CODE REVIEW

**Comment by li-jin-gou** [comment]
待 https://github.com/cloudwego/kitex/pull/456 合入,修改 `go.mod` 依赖

File: README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+		}
+	})
+	...
+}
+
+```
+
+### Screenshot
+
+<img src="images/meta.png" width="1674"  alt=""/>
```
Comment (GuangmingLuo) [inline README.md:59]: 这个图片展示的信息有点少啊，咱们上传的信息没法全部展示吗？

Comment (li-jin-gou) [reply README.md:59]: 对，我在补一个,那个 [opensergo-dashboard](https://github.com/opensergo/opensergo-dashboard) 页面展示的信息比较有限 :smiling_imp:

Comment (GuangmingLuo) [reply README.md:59]: 可以给他们提个需求，或者直接提PR，尽量补全一点

File: README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# opensergo (*This is a community driven project*)
+
+## metainfo
+Report meta info to opensergo
+
+### How to use?
+
+- run [opensergo-dashboard](https://github.com/opensergo/opensergo-dashboard)
+- report meta info need register start hook function
```
Comment (GuangmingLuo) [inline README.md]: Please give a detail example. e.g. env configuration and go run example.go

File: util/config.go [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+)
+
+type OpenSergoConfig struct {
+	Endpoint string `json:"endpoint"`
+}
+
+func GetOpenSergoConfig() (*OpenSergoConfig, error) {
+	// refer to https://github.com/opensergo/opensergo-specification/blob/main/specification/en/README.md
+	c := OpenSergoConfig{
+		Endpoint: os.Getenv("OPENSERGO_ENDPOINT"),
```
Comment (GuangmingLuo) [inline util/config.go]: where is the definition of "OPENSERGO_ENDPOINT"?

File: util/config.go [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+	}
+	if v := os.Getenv("OPENSERGO_BOOTSTRAP"); v != "" {
+		if err := json.Unmarshal([]byte(v), &c); err != nil {
+			return nil, err
+		}
+	}
```
Comment (GuangmingLuo) [inline util/config.go]: either of the two environment variables exists, it works and the method should return

File: README.md (hunk 2 above)
Comment (li-jin-gou) [reply README.md]: 收到

File: util/config.go (hunk 3 above)
Comment (li-jin-gou) [reply util/config.go]: 目前删除了这段逻辑

File: util/config.go (hunk 4 above)
Comment (li-jin-gou) [reply util/config.go]: done

File: README.md (hunk 1 above)
Comment (GuangmingLuo) [reply README.md:59]: 应该是少了 `listeningAddresses ` 和 `methods.inputTypes`, `methods.outputTypes` 信息的展示

**Review by GuangmingLuo — APPROVED** [review]

**Review by jayantxie — APPROVED** [review]

**Pull request merged**